src/Device.ts

```diff
@@ -51,9 +51,12 @@ import { CertificationDeclarationManager } from "./matter/certificate/Certificat
 const logger = Logger.get("Device");
 
 class Device {
+
+
     async start() {
         logger.info(`node-matter@${packageJson.version}`);
 
+        const networkInterface = getParameter("interface");
```

Comment:
can we please call it "announceInterface" or "broadcastInterface", so that it is more clear